core/src/main/java/com/linecorp/armeria/client/encoding/GzipStreamDecoderFactory.java

```diff
@@ -29,7 +30,7 @@ public String encodingHeaderValue() {
     }
 
     @Override
-    public StreamDecoder newDecoder() {
```

Comment:
this probably doesn't impact many folks, but this is an api break I think

Reply:
Ah, thanks! Added it in the release note.